src/sass/_migrate_instance.scss

```diff
@@ -0,0 +1,27 @@
+.migrate-instance-modal {
+  .migrate-instance-table {
+    min-height: 15rem;
```

Comment:
```suggestion
```
Seems like a leftover?

Reply:
yes that's correct, now we dynamically adjust the table height in JS, will remove